docs/ci-cd-environment/aws-support.md

```diff
@@ -96,18 +115,38 @@ After creating the AWS IAM policy, save the ARN.
 
 ### 4. Create the stack configuration
 
+The stack accepts a number of [parameters for configuration](#configuration). These parameters can be specified either with a configuration file or using environment variables.
+
+#### Create configuration for Linux agents
+
 Create a `config.json` file with the following:
 
 ```json
 {
   "SEMAPHORE_AGENT_STACK_NAME": "<your-stack-name>",
   "SEMAPHORE_AGENT_TOKEN_PARAMETER_NAME": "<your-ssm-parameter-name>",
   "SEMAPHORE_AGENT_TOKEN_KMS_KEY": "<your-ssm-parameter-name>",
-  "SEMAPHORE_ENDPOINT": "<your-organization>.semaphoreci.com"
+  "SEMAPHORE_ENDPOINT": "<your-organization>.semaphoreci.com",
```

Comment:
Will this be a valid JSON if the last one has a comma?

Reply:
🤦